iOS/README.rst

```diff
@@ -0,0 +1,392 @@
+        # The name of the .dylib file, relative to the install base
+        RELATIVE_DYLIB=${FULL_DYLIB#$CODESIGNING_FOLDER_PATH/$INSTALL_BASE/}
+        # The full dotted name of the binary module, constructed from the file path.
+        FULL_MODULE_NAME=$(echo $RELATIVE_DYLIB | cut -d "." -f 1 | tr "/" ".");
+        # A bundle identifier; not actually used, but required by Xcode framework packaging
+        FRAMEWORK_BUNDLE_ID=$(echo $PRODUCT_BUNDLE_IDENTIFIER.$FULL_MODULE_NAME | tr "_" "-")
+        # The name of the framework folder.
+        FRAMEWORK_FOLDER="Frameworks/$FULL_MODULE_NAME.framework"
+
+        # If the framework folder doesn't exist, create it.
+        if [ ! -d "$CODESIGNING_FOLDER_PATH/$FRAMEWORK_FOLDER" ]; then
+            echo "Creating framework for $RELATIVE_DYLIB"
+            mkdir -p "$CODESIGNING_FOLDER_PATH/$FRAMEWORK_FOLDER"
+
+            cp "$CODESIGNING_FOLDER_PATH/dylib-Info-template.plist" "$CODESIGNING_FOLDER_PATH/$FRAMEWORK_FOLDER/Info.plist"
+            plutil -replace CFBundleExecutable -string "$DYLIB" "$CODESIGNING_FOLDER_PATH/$FRAMEWORK_FOLDER/Info.plist"
+            plutil -replace CFBundleIdentifier -string "$FRAMEWORK_BUNDLE_ID" "$CODESIGNING_FOLDER_PATH/$FRAMEWORK_FOLDER/Info.plist"
+        fi
+
+        echo "Installing binary for $RELATIVE_DYLIB"
+        mv "$FULL_DYLIB" "$CODESIGNING_FOLDER_PATH/$FRAMEWORK_FOLDER"
+    }
+
+    PYTHON_VER=$(ls "$CODESIGNING_FOLDER_PATH/python/lib")
+    echo "Install Python $PYTHON_VER standard library dylibs..."
+    find "$CODESIGNING_FOLDER_PATH/python/lib/$PYTHON_VER/lib-dynload" -name "*.dylib" | while read FULL_DYLIB; do
+        install_dylib python/lib/$PYTHON_VER/lib-dynload "$FULL_DYLIB"
+    done
+
+    # Clean up dylib template
+    rm -f "$CODESIGNING_FOLDER_PATH/dylib-Info-template.plist"
+
+    echo "Signing frameworks as $EXPANDED_CODE_SIGN_IDENTITY_NAME ($EXPANDED_CODE_SIGN_IDENTITY)..."
+    find "$CODESIGNING_FOLDER_PATH/Frameworks" -name "*.framework" -exec /usr/bin/codesign --force --sign "$EXPANDED_CODE_SIGN_IDENTITY" ${OTHER_CODE_SIGN_FLAGS:-} -o runtime --timestamp=none --preserve-metadata=identifier,entitlements,flags --generate-entitlement-der "{}" \;
+
+9. Add Objective C code to initialize and use a Python interpreter in embedded
+   mode. When configuring the interpreter, you can use:
+
+      [NSString stringWithFormat:@"%@/python", [[NSBundle mainBundle] resourcePath], nil]
+
+   as the value of ``PYTHONHOME``; the standard library will be installed as the
+   ``lib/python3.X`` subfolder of that ``PYTHONHOME``.
+
+If you have third-party binary modules in your app, they will need to be:
+
+* Compiled for both on-device and simulator platforms;
+* Copied into your project as part of the script in step 9;
+* Installed and signed as part of the script in step 10.
+
+Testing Python on iOS
+=====================
+
+The ``Tools/iOSTestbed`` folder that contains an Xcode project that is able to run
```

Comment:
The Tools folder has a lot of other `testbench` code (like the WASM test harness); the only other analog I can think of is the top level `PCBuild` folder, which has the Visual Studio project files for Windows builds - but I believe that name/location is (very) historical. 

Ultimately, the location doesn't functionally matter - any location will work. I'll defer to the preferences of the core team.

Reply:
There's some inconsistency here already, with Mac and PC being at the top level, and WASM being under Tools. I don't have any preference for one over the other, but wherever it is, it would seem like a good idea to put all the iOS stuff in the same place.